### ISSUE TITLE
aria-expanded= missing on subpages

### ISSUE THREAD

**Issue by gilestaylor** [sniggle/joomla-bootstrap4-template#15]
aria-expanded="true" is added to the <a> on the home page but pages below aria-expanded= is missing from the tag. Making the link active reloading the page.

Any ideas are welcome.

Many thanks
Giles

**Comment by gilestaylor** [comment]
Installed the plugin as suggested which fixed the issue.
Many thanks

**Comment by jwiesel** [comment]
good to hear that. I'll close the ticket accordingly.
